core/client/fs/src/main/java/alluxio/client/file/cache/LRUCacheEvictor.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * LRU client-side cache eviction policy.
+ */
+public class LRUCacheEvictor implements CacheEvictor {
```

Comment:
is this class thread safe? If yes, please annotate it with `@ThreadSafe`; if not, currently `LocalCacheManager` does not provide any guard to protect concurrent access to this evictor from multiple threads.

Reply:
It is. Updated with annotation.